### ISSUE TITLE
Valve servo frequency may not work due to issue in code

### ISSUE THREAD

**Issue by MaSc82** [jcfain/TCodeESP32#18]
https://github.com/jcfain/TCodeESP32/blob/cdf70582ae4be0339ce85ea8df33f812e61b8519/ESP32/src/TCode/v1.0/ServoHandler1_0.h#L352C1-L352C107

**TwistServo_Int** here should be **ValveServo_Int**. This way the valve servo frequency seems not to be taken into account.

**Comment by jcfain** [comment]
Good catch but this code has been deleted as it was never working in the first place. Timer code has been refactored as well.
